src/app/forms/templates/test-records/master.template.ts

```diff
@@ -395,6 +404,16 @@ export const masterTpl: Record<VehicleTypes, Partial<Record<keyof typeof TEST_TY
       customDefects: CustomDefectsHiddenSection,
       reasonForCreation: reasonForCreationSection
     },
+    testTypesDeskBasedGroup2: {
+      required: DeskBasedRequiredSectionHgvTrl,
+      vehicle: DeskBasedVehicleSectionHgvGroup1And2And4,
+      test: AmendDeskBasedTestSectionGroup2,
+      visit: VisitSection,
+      notes: NotesSection,
+      defects: defectsHiddenSection,
+      customDefects: CustomDefectsHiddenSection,
```

Comment:
```suggestion
```
should still work (for all desk-based tests)

Reply:
It's on all the others so I left it 😢